src/app/client/src/app/app.module.ts

```diff
@@ -23,6 +23,9 @@ import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
 import { TranslateHttpLoader } from '@ngx-translate/http-loader';
 import { TranslateStore } from '@ngx-translate/core';
 import { SbSearchFilterModule } from '@project-sunbird/common-form-elements';
+import { UserOnboardingModule} from '../app/modules/user-onboarding';
```

Comment:
@rajeevsathish 
Do we need to keep this module at root level or shared-feature module ?